### PR TITLE
[Feature] Limit TechStack and Link entries to 15 per user (Fixes #25)

### LINKED ISSUE #25: [Feature] Limit TechStack and Link entries to 15 per user

### PR DESCRIPTION
**Pull request by tawaks**
**Main Changes: "Fixes #25"**
- Add count check in `createSkill` and `createLink` actions
- Return appropriate error messages when limit is reached

- Implement `isLimitReached`  variable in `SkillForm` and `LinkForm`
- Disable submit button when limit is reached
- Display limit reached message to users

### CODE REVIEW

**Comment by tawaks** [comment]
image
image

File: src/lib/components/MyProfile/SkillsForm.svelte [hunk 1]
```diff
@@ -81,5 +84,11 @@
 		</Form.Control>
 	</Form.Field>
 
-	<Form.Button>Add</Form.Button>
+	<Form.Button disabled = {isLimitReached}>Add</Form.Button>
 </form>
+
+{#if isLimitReached}
+  <p class="text-red-500 mt-2 text-center">You have reached the maximum limit of 15 links.</p>
```
Comment (s1lvax) [inline src/lib/components/MyProfile/SkillsForm.svelte:91]: Could you please changes this to "You have reached the maximum limit of 15 skills.".

Thank you !

**Review by s1lvax — CHANGES_REQUESTED** [review]
Hi there,

Welcome and thank you for contributing to the project !

Your PR is looking very good, there's just a small change I ask you to do before merging.


Thanks :)

File: src/lib/components/MyProfile/SkillsForm.svelte (hunk 1 above)
Comment (Butugey12) [reply src/lib/components/MyProfile/SkillsForm.svelte]: Hi , Thanks for the warm welcome.
I got the Fix in

Comment (tawaks) [reply src/lib/components/MyProfile/SkillsForm.svelte]: Hi Thanks for the warm Welcome :)
I got the fix in

**Comment by s1lvax** [comment]
Looking good, thanks again for your time :D.

**Pull request merged**